tests/mocks.ts

```diff
@@ -0,0 +1,187 @@
+export const artifacts: any = [];
+
+export const assets: any = [];
+
+export const workflowRun: WorkflowRun = {
+  id: 4309583450,
+} as any;
+
+export let inputs: { [key: string]: string } = {};
```

Comment:
This looks like we're maintaining and exposing some "global state" that's later set and read by individual tests. Is that true?

If it is, I'm wondering if this limits the ability for tests to run in parallel, and also, global state can sometimes lead to code being trickier to debug and harder to follow.

What are your thoughts?

Reply:
This could be a problem, let me see if I can relatively easily fix this.

Reply:
Ok, moved the state so it is no longer global, only per `getMocks` invocation. Each module is effectively executed in sequence in jest, so the module-level scope should not cause problems.